### DISCUSSION TITLE
4.1.8

### DISCUSSION THREAD

**Discussion by timmo001** [timmo001/system-bridge#3014] [General]
## 🐛 Bug fixes

- Update systembridgebackend to 4.2.1 @timmo001 (#3004)
- Update systembridgecli to 4.0.4 @timmo001 (#3005)
- Update systembridgebackend to 4.2.2 @timmo001 (#3007)
- Bump systembridgebackend from 4.2.2 to 4.2.3 @dependabot (#3012)

## 🚀 Enhancements

- Write current version to a file for later use from the backend @timmo001 (#3001)
- Update systembridgebackend to 4.2.0 @timmo001 (#3003)
- Update systembridgefrontend to 4.0.3 @timmo001 (#3006)
- Update and add new logs path to help menu @timmo001 (#3009)

## ⬆️ Dependency updates

- Update systembridgebackend to 4.2.0 @timmo001 (#3003)
- Update systembridgebackend to 4.2.1 @timmo001 (#3004)
- Update systembridgecli to 4.0.4 @timmo001 (#3005)
- Update systembridgefrontend to 4.0.3 @timmo001 (#3006)
- Update systembridgebackend to 4.2.2 @timmo001 (#3007)
- Bump sysinfo from 0.30.8 to 0.30.9 in /src-tauri @dependabot (#3011)
- Bump systembridgebackend from 4.2.2 to 4.2.3 @dependabot (#3012)
- Bump systembridgecli from 4.0.4 to 4.0.5 @dependabot (#3013)

<hr /><em>This discussion was created from the release <a href='https://github.com/timmo001/system-bridge/releases/tag/4.1.8'>4.1.8</a>.</em>

**Comment by Joka69** [comment]
Hi there,

Is there a way to change the localhost IP of a windows machine/build so it can pick up in HASS (Virtual box) i cannot connect to localhost, it just times out immediately.

When i try with IP it runs then times out? Im not sure where in the config im supposed to hange it to 0.0.0.0 so i can connect using my machine IP (Windows) 

![Capture](https://github.com/timmo001/system-bridge/assets/5106430/934c00a1-3b16-44c9-9084-d86dbc15bf7a)

**Reply by DeFlanko** [reply]
You put the IP (good time to setup either DNS or a static IP, too) of the Host running System Bridge in there. 

also, the Token is different if your running as a system account -- ie. running the app as a Scheduled Task with a service account. youll need to login the host machine with aht service account, run system bridge and get the token of that logged in user.

Its how i have mine setup. 

 
![image](https://github.com/timmo001/system-bridge/assets/5233969/d047b581-d490-44e7-9c56-825b7a3280fb)

![image](https://github.com/timmo001/system-bridge/assets/5233969/eadb69e1-8868-4335-bbb5-298a517bad23)


![image](https://github.com/timmo001/system-bridge/assets/5233969/cf717ddc-ad12-46fd-bc6e-7ef493370526)
